packages/main/src/Popover.js

```diff
@@ -447,6 +447,8 @@ class Popover extends Popup {
 		}
 		this.arrowTranslateY = Math.round(arrowY);
 
+		top = this._adjustForIOSKeyboard(top);
```

Comment:
I wonder if we should do it only when isPhone and skip it otherwise.

Reply:
generally it should be the case only for iOS devices (iPad and iPhone). But since the code is not heavy we decided to leave it for all.
Should we add a new check for isIOS in Device.js? and use it

Reply:
did it with iOS check